### PR TITLE
sys-boot/gnu-efi: bump to 3.0.12

### PR DESCRIPTION
**Pull request by kobboi**
Closes: https://bugs.gentoo.org/701152
Package-Manager: Portage-2.3.101, Repoman-2.3.22
Signed-off-by: Christophe Lermytte <gentoo@lermytte.be>

Keeping KEYWORDS as for 3.0.11, since stable efitools will not build without this.

### CODE REVIEW

File: sys-boot/gnu-efi/gnu-efi-3.0.12.ebuild [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Copyright 2004-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic toolchain-funcs
+
+DESCRIPTION="Library for build EFI Applications"
+HOMEPAGE="http://gnu-efi.sourceforge.net/"
```
Comment (charIes17) [inline sys-boot/gnu-efi/gnu-efi-3.0.12.ebuild:9]: curl --head http://gnu-efi.sourceforge.net/
HTTP/1.1 302 Found
Server: nginx/1.14.0 (Ubuntu)
Date: Mon, 22 Jun 2020 14:18:17 GMT
Content-Type: text/html
Content-Length: 0
Connection: keep-alive
Vary: Host
Location: https://sourceforge.net/projects/gnu-efi/
X-From: sfp-web-6
Vary: Accept-Encoding

**Comment by floppym** [comment]
Have you tested any reverse deps?

https://qa-reports.gentoo.org/output/genrdeps/dindex/sys-boot/gnu-efi

**Comment by kobboi** [comment]
@floppym 

**gnu-efi-3.0.12**

[NOK] app-crypt/efitools-1.7.0 [SSL related issue]
[OK ] app-crypt/efitools-1.8.1
[OK ] app-crypt/efitools-1.9.2
[NOK] app-crypt/pesign-113 [bug #721934]
[NOK] app-crypt/sbsigntools-0.6-r2 [SSL related issue]
[OK ] app-crypt/sbsigntools-0.9.1-r1
[OK ] app-crypt/sbsigntools-0.9.2
[OK ] sys-apps/fwupd-1.2.11
[OK ] sys-apps/fwupd-1.3.10
[OK ] sys-apps/fwupd-1.3.11
[OK ] sys-apps/fwupd-1.4.2
[OK ] sys-apps/fwupd-1.4.4
[OK ] sys-apps/systemd-244.3
[OK ] sys-apps/systemd-245-r5
[OK ] sys-apps/systemd-245.5
[NOK] sys-apps/systemd-9999
[OK ] sys-boot/elilo-3.16-r3
[NOK] sys-boot/refind-0.11.4 
[NOK] sys-boot/refind-0.12.0 [bug #723244]
[NOK] sys-boot/syslinux-6.03
[NOK] sys-boot/syslinux-6.04_pre1
[OK ] sys-boot/syslinux-6.04_pre1-r1
[OK ] sys-boot/syslinux-6.04_pre1-r2

**gnu-efi-3.0.11**

[NOK] app-crypt/efitools-1.7.0
[NOK] app-crypt/efitools-1.8.1
[NOK] app-crypt/efitools-1.9.2
[NOK] app-crypt/pesign-113
[NOK] app-crypt/sbsigntools-0.6-r2
[OK ] app-crypt/sbsigntools-0.9.1-r1
[OK ] app-crypt/sbsigntools-0.9.2
[OK ] sys-apps/fwupd-1.2.11
[OK ] sys-apps/fwupd-1.3.10
[OK ] sys-apps/fwupd-1.3.11
[OK ] sys-apps/fwupd-1.4.2
[OK ] sys-apps/fwupd-1.4.4
[OK ] sys-apps/systemd-244.3
[OK ] sys-apps/systemd-245-r5
[OK ] sys-apps/systemd-245.5
[NOK] sys-apps/systemd-9999
[OK ] sys-boot/elilo-3.16-r3
[NOK] sys-boot/refind-0.11.4
[NOK] sys-boot/refind-0.12.0
[NOK] sys-boot/syslinux-6.03
[NOK] sys-boot/syslinux-6.04_pre1
[OK ] sys-boot/syslinux-6.04_pre1-r1
[OK ] sys-boot/syslinux-6.04_pre1-r2

**Comment by floppym** [comment]
Thanks! I don't see any regressions.

**Pull request closed without merging**